controlsfx/src/main/java/impl/org/controlsfx/skin/RatingSkin.java

```diff
@@ -259,14 +259,17 @@ private void updateClip() {
     	
     }
     
-//    private double getSpacing() {
-//        return (backgroundContainer instanceof HBox) ?
-//                ((HBox)backgroundContainer).getSpacing() :
-//                ((VBox)backgroundContainer).getSpacing();
-//    }
+    /**
+     * Creates the node for one star.
+     * By default, an empty region is returned.
+     * This method can be overwritten to customize the styling of the star, e.g. to use a font-based icon.
+     */
+    protected Node createButtonNode() {
```

Comment:
A more generic name for this method is "display node" instead of "button node" i.e. `createDisplayNode`. Although we use the style-class "button", in theory it is not a button.

Reply:
I agree with you. However, it reacts as some form of button (has onclick action) and is called a button everywhere else in code.

Reply:
I would like @Maxoudela to pitch in his view.

Reply:
Sorry for the delay.
The button naming is maybe a bit wrong but it's indeed called "buttons" everywhere in the file so I would say to keep it that way for consistency.